### DISCUSSION TITLE
UCI - Netcore WA API Credentials

### DISCUSSION THREAD

**Discussion by aayushkedia95** [Samagra-Development/SamagraX-community#32] [Installation and Configuration]
@vinukumar-vs @vrayulu, can you share the DIKSHA Netcore WhatsApp API credentials with us? 

We require them for creating the Netcore WhatsApp Adapter for UCI. Would be great if this can be shared at the earliest. 
As of today, we have this documentation with us - https://wadocs.pepipost.com/

cc- @Krishnaj20 @ChakshuGautam

**Comment by aayushkedia95** [comment]
Also, sharing our callback URL for reference - http://comms.samagra.io/netcore/whatsApp

**Reply by aayushkedia95** [reply]
@vinukumar-vs, it is possible to get on a call with you and Netcore team tomorrow first half? 
We wanted some clarifications on Netcore APIs. 

cc- @Krishnaj20 @ChakshuGautam

**Reply by vinukumar-vs** [reply]
@aayushkedia95 For any discussions with Netcore, we have to communicate with @ronish(Netcore support)  through emails. I will loop you in the discussion we are having with Ronish on dev & staging env setup.

@Nik720 @Krishnaj20

**Reply by aayushkedia95** [reply]
Okay @vinukumar-vs. That would be very helpful.
